.circleci/config.yml

```diff
@@ -646,6 +646,8 @@ jobs:
           environment:
               TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_CUDA: true
               TORCHAUDIO_TEST_ALLOW_SKIP_IF_ON_PYTHON_310: true
+              TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_DEVICE: true
+              TORCHAUDIO_TEST_ALLOW_SKIP_IF_NO_MACOS: true
```

Comment:
`config.yml` is auto-generated from `config.yml.in`. Can you apply the same change to `config.yml.in` and run `regenerate.py` then check-in both `config.yml` and `config.yml.in`?